statistic-labels/index.html

```diff
@@ -338,6 +339,11 @@ <h3 id="printBasinTypeHeader">Basin Characteristic Types</h3>
 			  <select onChange="filterByStatGroup(this.value)" id="statTypeDropdown">
 				<option value="None">None</option>
 			  </select>
+			  <br>
```

Comment:
Add an additional <br> between the drop-downs for aesthetics 
![image](https://user-images.githubusercontent.com/40237491/147692898-d4cc1972-1ea2-4715-b244-730c7748f192.png)